flink-python/pyflink/table/tests/test_window.py

```diff
@@ -40,8 +41,8 @@ class BatchTableWindowTests(PyFlinkBatchTableTestCase):
 
     def test_tumble_window(self):
         t = self.t_env.from_elements([(1, 1, "Hello")], ["a", "b", "c"])
-        result = t.window(Tumble.over("2.rows").on("a").alias("w"))\
-            .group_by("w, c").select("b.sum")
+        result = t.window(Tumble.over(E.lit(2).rows).on("a").alias("w"))\
```

Comment:
".rows" seems not supported. Maybe we should support it or use "row_interval()" instead?